src/main/java/com/github/mauricioaniche/ck/util/JDTUtils.java

```diff
@@ -76,13 +127,24 @@ public static String getMethodFullName(MethodDeclaration node) {
 			}
 		}
 
-		return String.format("%s/%d%s%s%s",
-				methodName,
+		return String.format("%d%s%s%s",
```

Comment:
Can we remove the `String.format(...)` duplication that I see here and below?

Reply:
Yes